src/main/drivers/vtx_var.c

```diff
@@ -0,0 +1,26 @@
+#include <stdint.h>
+#include "vtx_var.h"
+
+const uint16_t vtx58FreqTable[5][8] =
+{
+    { 5865, 5845, 5825, 5805, 5785, 5765, 5745, 5725 }, // Boscam A
+    { 5733, 5752, 5771, 5790, 5809, 5828, 5847, 5866 }, // Boscam B
+    { 5705, 5685, 5665, 5645, 5885, 5905, 5925, 5945 }, // Boscam E
+    { 5740, 5760, 5780, 5800, 5820, 5840, 5860, 5880 }, // FatShark
+    { 5658, 5695, 5732, 5769, 5806, 5843, 5880, 5917 }, // RaceBand
+};
+
+const char * const vtx58BandNames[] = {
+    "--------",
+    "BOSCAM A",
+    "BOSCAM B",
+    "BOSCAM E",
+    "FATSHARK",
+    "RACEBAND",
+};
+
+const char vtx58BandLetter[] = "-ABEFR";
+
+const char * const vtx58ChanNames[] = {
+    "-", "1", "2", "3", "4", "5", "6", "7", "8",
+};
```

Comment:
Bandnames and Channelnames belong in the `io` directory, not the `drivers` directory.

Also please don't use unnecessary abbreviations. `vtx58ChannelNames` is better than `vtx58ChanNames` - code is read much more than it written.